src/DripsHub.sol

```diff
@@ -207,14 +205,13 @@ contract DripsHub is Managed, Drips, Splits {
     /// If too low, receiving will be cheap, but may not cover many cycles.
     /// If too high, receiving may become too expensive to fit in a single transaction.
     /// @return receivedAmt The received amount
-    /// @return receivableCycles The number of cycles which still can be received
     function receiveDrips(uint256 userId, IERC20 erc20, uint32 maxCycles)
         public
         whenNotPaused
-        returns (uint128 receivedAmt, uint32 receivableCycles)
+        returns (uint128 receivedAmt)
```

Comment:
I think it makes sense. As you said the user could use `receivableDripsCycles` to get the cycles. 
It is also kind of an edge-case, that a user needs this feature. 

So it makes sense to remove the information for the `receiveDrips` :+1: